addons/web_editor/static/lib/odoo-editor/test/spec/copyPaste.test.js

```diff
@@ -1907,4 +1908,45 @@ describe('Copy and paste', () => {
             });
         });
     });
+    describe('HTML tables', async () => {
+        describe('From Microsoft Excel Online', async () => {
+            it('should keep all allowed style', async () => {
+                const result = await fetchFileContents('/web_editor/static/lib/odoo-editor/test/spec/html_tables/excel_odoo.html');
+                await testEditor(BasicEditor, {
+                    contentBefore: '<p>[]</p>',
+                    stepFunction: async editor => {
+                        const clipboard = await fetchFileContents('/web_editor/static/lib/odoo-editor/test/spec/html_tables/excel_clipboard.html');
+                        await pasteHtml(editor, clipboard);
+                    },
+                    contentAfter: result,
```

Comment:
Just so you know you can in fact `await` inline here, no need for a local variable. The reason I say this is because I find it a little disconcerting to see the expected result before the actual test when reading the file. Is it only me ?

Reply:
It's not just you, I just didn't figure out what you suggest.